XamlControlsGallery/ControlPagesSampleCode/ListView/ListViewSample4_cs.txt

```diff
@@ -0,0 +1,49 @@
+// C# Code
+
+// Make sure to add this to your declarations at the top for the filtering method:
+using System.Linq;
+// ...
+
+// Define an ObservableCollection<Contact> object to serve as the ListView's
+// ItemsSource. This collection will get updated after the filters are used:
+ObservableCollection<Contact> contactsFiltered = new ObservableCollection<Contact>();
+
+/* Populate contactsFiltered with all contact objects (in this case, contacts holds
+all of our Contact objects so we copy them into contactsFiltered). Set this newly populated
+collection as the ItemsSource for the ListView. */
+contactsFiltered = new ObservableCollection<Contact>(contacts3);
```

Comment:
`contacts3` comes out of nowhere. Maybe we can have a list "allContacts" as a property. If we do this, we can also provide implementations for the methods `AddBack_Contacts` and `Remove_NonMatching`.

Either way, maybe `allContacts` would be a better name instead of `contacts3` I think.

Reply:
Yeah, I think I was just copying straight from the source code, I'll change it to allContacts 👍